files/guides/qol.md

```diff
@@ -0,0 +1,28 @@
+# Credits
+
+#### Source
```

Comment:
You should probably make these links look nice by using this syntax: `[text](url "hover text)`